admin/settings/advanced.mdx

```diff
@@ -39,11 +39,12 @@ To disable the collection of file-based logs, navigate to Settings, select Advan
 
 # Metrics Ingestion
 
-By default, MW Agent metrics are automatically ingested. To disable this feature, press the radio button and click Restart Agent.
+By default, MW Agent metrics are automatically ingested. To disable this feature, press the radio button and click Save.
```

Comment:
Suggestion to add a sentence on what impact disabling metrics (and traces) ingestion has on the system.